packages/create-svelte/cli/index.js

```diff
@@ -103,25 +108,25 @@ async function main() {
  * @param {string} target
  */
 async function prompt_modifications(target) {
+	const template_response = await prompts({
+		type: 'select',
+		name: 'value',
+		message: 'Which app Svelte template do you want to use?',
+		choices: [
+			{ title: 'Default App (Counter + Route)', value: 'default' },
+			{ title: 'Skeleton App', value: 'skeleton' }
+		]
+	});
+	const is_skeleton_template = template_response.value === 'skeleton';
+	await add_skeleton_template(target, is_skeleton_template);
+
 	const ts_response = await prompts({
 		type: 'confirm',
 		name: 'value',
 		message: 'Use TypeScript in components?',
 		initial: false
 	});
-	await add_typescript(target, ts_response.value);
-
-	const css_response = await prompts({
```

Comment:
Personally I would have left that in it's really easy to setup, we don't even need to add much code modifications. Also, do we have any data how many people would choose what? I think people would opt for scss quite a bit just because many component libraries use it for their styling/theming.

Reply:
Hm, I see your point.. The only drawback that I see is that we would need to maintain 3 different styling files (I believe that's why @benmccann suggested removing it) but i's not a big deal for me -- it'd be easy to revert it.

@dummdidumm @benmccann How I can move forward with it?

Reply:
We chatted about this part for a bit and it sounds like @Rich-Harris and @pngwn also vote to take out the CSS stuff